catalog/dags/providers/provider_dag_factory.py

```diff
@@ -594,17 +656,26 @@ def create_day_partitioned_reingestion_dag(
             # This gates the tasks at each level.
             gather_operator >> partitioned_ingest_workflows[i + 1]
 
+        report_aggregate_errors = report_aggregate_reingestion_errors(provider_conf)
+
         # Create a single report_load_completion task, passing in the list of duration
         # and counts data for each completed task.
         report_load_completion = create_report_load_completion(
-            conf.dag_id, conf.media_types, ingestion_metrics, conf.dated
+            provider_conf.dag_id,
+            provider_conf.media_types,
+            ingestion_metrics,
+            provider_conf.dated,
         )
 
         # report_load_completion is downstream of all the ingestion TaskGroups in the
         # final list.
-        partitioned_ingest_workflows[-1] >> report_load_completion
+        (
+            partitioned_ingest_workflows[-1]
+            >> report_aggregate_errors
```

Comment:
It will not double the reports in normal ingestion DAGs, right?

Reply:
Correct! This task is only appended to reingestion workflows, so there should be zero change to normal ingestion.